libs/assertion/include_compatibility/hpx/util/assert.hpp

```diff
@@ -0,0 +1,26 @@
+//  Copyright (c) 2013 Antoine Tran Tan
+//  Copyright (c) 2001, 2002 Peter Dimov and Multi Media Ltd.
+//  Copyright (c) 2007 Peter Dimov
+//  Copyright (c) Beman Dawes 2011
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+//  Make HPX inspect tool happy: hpxinspect:noassert_macro
+//                               hpxinspect:noinclude:HPX_ASSERT
+//                               hpxinspect:nodeprecatedname:BOOST_ASSERT
```

Comment:
Are those comments still needed?

Reply:
In this specific file, we don't really mind because it will be removed soon (just for compatibility of headers), but I will check if they are needed in `libs/assertion/include/hpx/assertion.hpp`